particles/Greet/Greet.js

```diff
@@ -0,0 +1,72 @@
+// @license
+// Copyright (c) 2017 Google Inc. All rights reserved.
+// This code may only be used under the BSD style license found at
+// http://polymer.github.io/LICENSE.txt
+// Code distributed by Google as part of this project is also
+// subject to an additional IP rights grant found at
+// http://polymer.github.io/PATENTS.txt
+
+"use strict";
+
+// Defines the Greet particle in-line. Every DOM particle
+// needs to call the defineParticle function to register itself.
+defineParticle(({DomParticle}) => {
+
+  // Specifies the DOM template to be used when the rendering function
+  // is called on the Greet particle. The template syntax is similar
+  // to the one used by web component templates.
+  const template = `
+    <style>
+      [hello] {
+        font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+        font-size: 16px;
+        background-color: #AED581;
+        padding: 20px;
+      }
+    </style>
+    <div hello>
+      <!-- Basic variable substitution where {{message}} will be replaced
+           by the value associated with 'message' when the particle is
+           rendered. Note that the span is necessary here because (for now)
+           variable substitution only works if the variable is a
+           lonely child of an element. -->
+      <div>
+        <span>{{message}}</span>!
+        <!-- Defines where and how the 'customgreeting' slot should be rendered
+             in the Greet particle DOM. Another particle may render content
+             into that slot. In our demo, the PersonalGreet particle
+             is rendering something into that slot. -->
```

Comment:
Should there be a note here about how our demo shows `PersonalGreet`, but in practice it could be another particle entirely?